include/copc-lib/las/file.hpp

```diff
@@ -1,34 +1,28 @@
 #ifndef COPCLIB_LAS_FILE_H_
 #define COPCLIB_LAS_FILE_H_
 
-#include <istream>
+#include <iostream>
 #include <map>
 
+#include <copc-lib/las/header.hpp>
 #include <copc-lib/las/vlr.hpp>
 
-#include <lazperf/readers.hpp>
-
 namespace copc::las
 {
-using VlrHeader = lazperf::vlr_header;
-// todo: create our own lasheader class so we can switch between header12/14
-using LasHeader = lazperf::header12;
-
 class LasFile
 {
   public:
-    LasFile(std::istream &in_stream);
+    LasFile(LasHeader header, las::EbVlr eb) : header_(header), eb(eb){};
 
-    std::vector<Vlr> vlrs;
-    LasHeader GetLasHeader() { return header_; }
+    std::map<uint64_t, las::VlrHeader> vlrs; // maps from absolute offsets to VLR entries
```

Comment:
It gets initialized here https://github.com/RockRobotic/copc-lib/blob/e9a2ad2a7b0b7547f1edc71e2e3872b919379998/src/io/reader.cpp#L29 although outside of that it isn't actually used. I'm okay with keeping it though if you are, it may be useful to a client. And it may be used in the future.

Reply:
Yep, we can keep it, just wanted to make sure we had a reason and wasn't just forgotten :)